sds_data_manager/constructs/ialirt_processing_construct.py

```diff
@@ -81,7 +81,7 @@ def create_ecs_security_group(self):
 
         # Allow inbound and outbound traffic from a specific port and IP.
         # IPs: LASP IP, BlueNet (tlm relay)
-        ip_ranges = ["128.138.131.0/24", "198.118.1.14/32"]
+        ip_ranges = ["128.138.131.0/24", "198.118.1.14/32", "193.174.22.3/32"]
```

Comment:
I'd suggest adding comments inline to the IPs you're using. You have them above for the previous two, but it might be easier to bring these onto separate lines and add comments next to them to make it clear.
```suggestion
        ip_ranges = ["128.138.131.0/24",  # LASP IP
                              "198.118.1.14/32",  # BlueNet
                              "193.174.22.3/32"]  # Kiel
```